requirements.txt

```diff
@@ -6,7 +6,7 @@ scipy>=0.18
 tabulate>=0.7
 scikit-learn
 tqdm
-tensorflow==2.2
-tfa-nightly==0.12.0.dev20200807032152
+tensorflow==2.3
```

Comment:
> Fixed. I supposed you wanted to be specific for such a core component (like with tfa-nightly).
> Do you also want to generalize tfa-nightly and set it to `tfa-nightly>=0.12.0,<0.13.0` ? (it will probably drop 2.2 support by then, looking at their current support matrix here https://github.com/tensorflow/addons and we could always expand that later)

Fixed ... yeah... sure... but did you know that committing without pushing has an interesting result? :)
Actually fixed and pushed :)